portfolio/src/main/webapp/script.js

```diff
@@ -114,10 +118,17 @@ function deleteComments() {
 }
 
 /** Creates an <li> element containing text. */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function createListTextElement(text) {
+    const liElement = document.createElement('li');
+    liElement.innerText = text;
+    return liElement;
+}
+
+/** Creates an <li> element containing an image. */
+function createListImageElement(imageURL) {
+    const liElement = document.createElement('li');
+    liElement.innerHTML = '<img src="' + imageURL + '" class="list-image">';
```

Comment:
can you use createElement and set the properties instead of string interpolation?

Reply:
yup will do!